package.json

```diff
@@ -9,7 +9,6 @@
     "build:packages": "lerna run build --no-private",
     "build:site": "lerna run build --scope=create-react-app",
     "postinstall": "lerna bootstrap --hoist --no-ci && npm run build:packages",
-    "tsd": "lerna run tsd",
```

Comment:
this command assumes that there's a tsd script in at least some of the monorepo package.json files. we should remove it from there too I guess

Reply:
Yes, I've checked all of them and there weren't in any of the packages...

Reply:
then this command was already doing nothing :D https://github.com/nearform/graphql-hooks/actions/runs/2950012855/jobs/4714237058#step:8:12